shell/completion.bash

```diff
@@ -31,9 +31,6 @@ if [[ $- =~ i ]]; then
 
 ###########################################################
 
-# To redraw line after fzf closes (printf '\e[5n')
```

Comment:
Can we keep this comment? It's not the most obvious part of the code. Let's put it above the first one.

Reply:
Done!